### DISCUSSION TITLE
ARM support?

### DISCUSSION THREAD

**Discussion by hexclann** [plausible/analytics#2191] [Self-Hosted Support]
Does plausible analytics docker installation supports ARM architecture?

**Comment by DennisLoska** [comment]
@hexclann Not officially yet, but it's working for me with this image https://github.com/plausible/analytics/pull/2103

**Comment by ruslandoga** [comment]
👋

[`v1.5.0-rc.0`](https://github.com/plausible/analytics/discussions/2324) is out with arm64 support! Please try it out :)
